menpo/image/base.py

```diff
@@ -167,7 +167,7 @@ def denormalize_pixels_range(pixels, out_dtype):
     return (pixels * max_range).astype(out_dtype)
 
 
-def channels_to_back(pixels):
+def roll_channels(pixels):
```

Comment:
@JeanKossaifi I promise. Last thing 😆 ! Since we settled on `channels_to_back` can we keep this as `channels_to_back`?